### ISSUE TITLE
jfrog rt bag results in "unquoted '\' must be followed by new line"

### ISSUE THREAD

**Issue by rpetti** [jfrog/jfrog-cli#1039]
**Describe the bug**
Running `jfrog rt bag <build> <buildnumber> <dir>` on Windows results in:
```
[Info] Collecting git branch, revision and remote url...
[Error] 15:14: unquoted '\' must be followed by new line
```

**To Reproduce**
- Clone a git repo on Windows
- Set sslCAInfo for the repo's host to some file on the system
```
git config --local --add http.https://github.com.sslCAInfo E:\Some\CA\Path
```
- Run the bag command
```
jfrog rt bag test 1 .
```

_Full example:_
```
git clone https://github.com/rpetti/rpetti.git
cd rpetti
git config --local --add http.https://github.com.sslCAInfo E:\Some\CA\Path
jfrog rt bag test 1 .
```

**Expected behavior**
We expect the command to work as advertised.

**Versions**
- JFrog CLI version: 1.45.1
- JFrog CLI operating system: Windows 10
- Artifactory Version: Artifactory 7.7.3 Enterprise X

**Additional context**
GitLab sets this git property on checkout, and it works fine. The JFrog CLI just can't seem to parse it correctly on windows (it works fine on Linux).

**Comment by sophietait** [comment]
When testing this issue, I found that the `jfrog rt bag` command was successful after running the `git config` command.

I initialized a git repository, instead of cloning a git repository, but followed the rest of the reproduction steps listed above. 
I tested with JFrog CLI version 1.44.0 on a Windows server.

<img width="1641" alt="Screen_Shot_2021-04-01_at_2_14_59_PM" src="https://user-images.githubusercontent.com/16872799/113601359-1282fd80-95f6-11eb-9fdd-918f8f55949b.png">
<img width="586" alt="Screen_Shot_2021-04-01_at_11_50_38_AM" src="https://user-images.githubusercontent.com/16872799/113601363-131b9400-95f6-11eb-8b95-3908e3aa12b1.png">

**Comment by rpetti** [comment]
@sophietait Please see my updated example.

**Comment by sophietait** [comment]
@rpetti reproduced the issue with an updated example.  Please update reproduction steps with the following line:

`git config --local --add http.https://github.com.sslCAInfo E:\\Some\\CA\\Path`

**Comment by rpetti** [comment]
I was able to reproduce it with single slashes in the command.

**Comment by abowergroup** [comment]
I am experiencing the same error on a number of commands. We just updated to the latest version of the CLI (1.46.1). Now, a number of scripted commands return the following error: [Error] 12:8: unquoted '\' must be followed by new line

The error occurs on the following commands:

jfrog rt upload "DasClientCore\bin\Debug\DasClientCore.1.1.0.nupkg" "nuget-local/IT/DasClientCore//" --build-name="DasClientCore" --module="DasClientCore" --build-number=8 --server-id="artifactory-dev"

jfrog rt build-add-git "DasClientCore" 8 --server-id="artifactory-dev"

**Comment by gailazar300** [comment]
Fixed in JFrog CLI version: 1.47.2.
